sirepo/package_data/static/js/sirepo.js

```diff
@@ -2242,6 +2242,16 @@ SIREPO.app.factory('requestSender', function(cookieService, errorService, $http,
         );
     };
 
+    self.statelessCompute = function(appState, data, callback) {
+        const m = SIREPO.APP_SCHEMA.constants.methods.statelessCompute;
```

Comment:
I don't think this is a good idea. Maintaining a list of methods in three places is unnecessary. It will be a programming error if the method is incorrect. An assertion on the server is all we need. (The issue of how that assertion is displayed is distinct and as you noted in our call this week, problematic currently, but that's a different bug.) Consider the other problem of maintaining a list of correct parameters, which is also important, and also a programming error that the server should assert.

Reply:
Ok. I waffled on this quite a bit. I had a feeling you might not like it:) I liked having everything enumerated but I suppose a simple grep in template will do the same.

d6fccd452